test/integration/test_cosign_real.sh

```
@@ -0,0 +1,60 @@
+QUAY_HOST="${QUAY_HOST:-}"
+QUAY_NAMESPACE="${QUAY_NAMESPACE:-}"
+QUAY_REPO="${QUAY_REPO:-cosign-verify}"
+IMAGE_TAG="${IMAGE_TAG:-latest}"
+
+if [[ -z "${QUAY_HOST}" || -z "${QUAY_NAMESPACE}" ]]; then
+  echo "QUAY_HOST and QUAY_NAMESPACE must be set" >&2
+  exit 1
+fi
+
+IMAGE_REF="${QUAY_HOST}/${QUAY_NAMESPACE}/${QUAY_REPO}:${IMAGE_TAG}"
+REGISTRY_REF="${QUAY_HOST}/${QUAY_NAMESPACE}/${QUAY_REPO}"
```

Shellcheck (0.11.0)

[warning] REGISTRY_REF appears unused. Verify use (or export if used externally).

(SC2034)